### PR TITLE
Basic styling

### PR DESCRIPTION
**Pull request by richardolsson**
In this PR, I have created some basic styling for the index.html document, added a nice red background color, and styled the `em` elements to be uppercase and bold.

### CODE REVIEW

File: main.css [hunk 1]
```diff
@@ -0,0 +1,9 @@
+body {
+    background-color: red;
```
Comment (saeraa) [inline main.css]: Jag tycker bakgrunden ska vara rosa.

File: index.html [hunk 2]
```diff
@@ -2,6 +2,7 @@
 <html>
     <head>
         <title>Git workflow experiment</title>
+        <link rel="stylesheet" type="text/css" href="main.css">
```
Comment (richardolsson) [inline index.html:5]: I like that we are using `link` and not `@import`, which is another way to include CSS from an external file.

File: main.css [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+    color: white;
```
Comment (richardolsson) [inline main.css]: This is completely illegible. Please invert it or something. I know you love red, but maybe not the background? ❤️

File: main.css [hunk 4]
```diff
@@ -0,0 +1,9 @@
...
+}
+
+em {
+    font-weight: bold;
```
Comment (richardolsson) [inline main.css:7]: I like this! 🎉

**Review by richardolsson — COMMENTED** [review]
This is my review. I have some changes I'd like you to make. See below.

**Pull request merged**